apps/internal/base/base.go

```diff
@@ -257,6 +268,72 @@ func (b Client) AcquireTokenByAuthCode(ctx context.Context, authCodeParams Acqui
 	return b.AuthResultFromToken(ctx, authParams, token, true)
 }
 
+// AcquireTokenByInteractive initiates the interactive auth flow.
+func (b Client) AcquireTokenByInteractive(ctx context.Context, params AcquireTokenInteractiveParameters) (AuthResult, error) {
```

Comment:
This is strictly a public client concern right? Is there a reason this can't live in the public package then?

Reply:
I emulated the other APIs as they appear to perform the heavy lifting inside the base client.  I can certainly move it (and dependencies) around if that's preferred.

Reply:
fixed